src/StatusQ/Components/StatusListItem.qml

```diff
@@ -13,8 +13,6 @@ Rectangle {
     property string titleId: ""
     property string title: ""
     property string titleAsideText: ""
-    property bool titleIcon1Visible
-    property bool titleIcon2Visible
```

Comment:
This is a breaking change, maybe that informationn can be added to the commit message in the footer so we'll have it show up in our changelogs?